### ISSUE TITLE
Sanity workflow collection installation fails on Ansible 2.9

### ISSUE THREAD

**Issue by kerry-meyer** [ansible-network/github_actions#104]
We are seeing a new failure that seems to be something that needs be fixed by the Ansible support group.

### Workflow:

The failure occurs when executing the following workflow:

```
jobs:
  sanity:
    uses: ansible-network/github\_actions/.github/workflows/sanity.yml@main
```

### Failure: (for Ansible core version 2.9)

Run ansible-galaxy collection install ./dellemc-enterprise_sonic-2.2.0.tar.gz --pre -p /home/runner/collections
ansible-galaxy collection install ./dellemc-enterprise_sonic-2.2.0.tar.gz --pre -p
*
*
*
ansible-galaxy: error: unrecognized arguments: --pre

### Details:
Run ansible-galaxy collection install ./dellemc-enterprise_sonic-2.2.0.tar.gz --pre -p /home/runner/collections
ansible-galaxy collection install ./dellemc-enterprise_sonic-2.2.0.tar.gz --pre -p /home/runner/collections
shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
env:
PY_COLORS: 1
source_directory: ./source
pythonLocation: /opt/hostedtoolcache/Python/3.7.17/x64
PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.7.17/x64/lib/pkgconfig
Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.7.17/x64
Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.7.17/x64
Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.7.17/x64
LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.7.17/x64/lib:/opt/hostedtoolcache/Python/3.9.17/x64/lib
usage: ansible-galaxy [-h] [--version] [-v] TYPE ...
ansible-galaxy: error: unrecognized arguments: --pre
Error: Process completed with exit code 2.

Sanity for Ansible 2.9 test matrix entries fails as shown above.

Will someone be deleting Ansible 2.9 soon from the test matrix in "https://github.com/ansible-network/github_actions/blob/main/.github/workflows/sanity.yml" since it has reached EOL?

**Comment by NilashishC** [comment]
@kerry-meyer A fix for this is being worked on in #103

**Comment by kerry-meyer** [comment]
Although I see that the fix coded in https://github.com/ansible-network/github_actions/pull/103 has been merged, the same failure is still occurring for sanity Ansible version 2.9 entries in the sanity test matrix.

Is some additional action needed to make the fix operational for sanity workflow executions that use the ".github/workflows/sanity.yml" workflow from the "https://github.com/ansible-network/github_actions" repo?
